proto/buf/alpha/registry/v1alpha1/studio_request.proto

```diff
@@ -77,7 +77,7 @@ message CreateStudioRequestRequest {
 // CreateStudioRequestResponse returns the created StudioRequest.
 message CreateStudioRequestResponse {
   // Created Studio Request.
-  StudioRequest request = 1;
+  StudioRequest created_request = 1;
```

Comment:
why not name them both `studio_request`

Reply:
Given it's already typed (we already know it's a `StudioRequest`), thought it would give more useful context to named them after the operation.